### PR TITLE
doc: update CHANGELOG for v2.2.0 release

### PR DESCRIPTION
**Pull request by coryan**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9763)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -141,6 +141,24 @@ the APIs in these libraries are stable, and are ready for production use.
 
 </details>
 
+### [Bigtable](/google/cloud/bigtable/README.md)
+
+- fix(bigtable): `DataConnection` refreshes channels ([#9718](https://github.com/googleapis/google-cloud-cpp/pull/9718))
+- doc: fix some broken links exposed by the use of mdformat ([#9725](https://github.com/googleapis/google-cloud-cpp/pull/9725))
```
Comment (dbolduc) [inline CHANGELOG.md:147]: delete

File: CHANGELOG.md [hunk 2]
```diff
@@ -141,6 +141,24 @@ the APIs in these libraries are stable, and are ready for production use.
...
+- fix(bigtable): Use retry policy on all streams with failing mutations ([#9706](https://github.com/googleapis/google-cloud-cpp/pull/9706))
+- feat(bigtable): per-operation Options ([#9627](https://github.com/googleapis/google-cloud-cpp/pull/9627)) ([#9623](https://github.com/googleapis/google-cloud-cpp/pull/9623))
+
+### [Dataproc](/google/cloud/dataproc/README.md)
+
+- feat(dataproc): mark the dataproc services as location dependent ([#9722](https://github.com/googleapis/google-cloud-cpp/pull/9722))
+
+### [Spanner](/google/cloud/spanner/README.md)
+
+- doc: fix some broken links exposed by the use of mdformat ([#9725](https://github.com/googleapis/google-cloud-cpp/pull/9725))
```
Comment (dbolduc) [inline CHANGELOG.md:157]: delete

File: CHANGELOG.md [hunk 3]
```diff
@@ -141,6 +141,24 @@ the APIs in these libraries are stable, and are ready for production use.
...
+- feat(spanner): fine-grained access control ([#9669](https://github.com/googleapis/google-cloud-cpp/pull/9669))
+- doc: add table of contents to landing page ([#9671](https://github.com/googleapis/google-cloud-cpp/pull/9671))
```
Comment (dbolduc) [inline CHANGELOG.md:159]: delete

File: CHANGELOG.md [hunk 4]
```diff
@@ -151,6 +169,18 @@ the APIs in these libraries are stable, and are ready for production use.
   `GOOGLE_CLOUD_CPP_STORAGE_USE_LEGACY_HTTP` to any value to use the legacy
   implementation. We are planning to remove the legacy code and the
   `GOOGLE_CLOUD_CPP_STORAGE_USE_LEGACY_HTTP` environment variable by 2022-12.
+- feat(storage): improve error messages on stalled uploads ([#9744](https://github.com/googleapis/google-cloud-cpp/pull/9744))
+- doc(storage): update broken docs link ([#9738](https://github.com/googleapis/google-cloud-cpp/pull/9738))
+- doc: fix some broken links exposed by the use of mdformat ([#9725](https://github.com/googleapis/google-cloud-cpp/pull/9725))
```
Comment (dbolduc) [inline CHANGELOG.md:174]: I'd delete these

File: CHANGELOG.md [hunk 5]
```diff
@@ -151,6 +169,18 @@ the APIs in these libraries are stable, and are ready for production use.
...
+- feat(storage): equality for Native IAM types ([#9649](https://github.com/googleapis/google-cloud-cpp/pull/9649))
+- doc(storage): fix comments for Notification requests ([#9642](https://github.com/googleapis/google-cloud-cpp/pull/9642))
```
Comment (dbolduc) [inline CHANGELOG.md:176]: Looks like these are internal comments. Delete.

File: CHANGELOG.md [hunk 6]
```diff
@@ -151,6 +169,18 @@ the APIs in these libraries are stable, and are ready for production use.
...
+- feat(storage): SA credentials default to self-signed JWTs ([#9629](https://github.com/googleapis/google-cloud-cpp/pull/9629))
+
+### [Common Libraries](/google/cloud/README.md)
+
+- feat: group GUAC `Options` in an `OptionsList` ([#9643](https://github.com/googleapis/google-cloud-cpp/pull/9643))
+- fix(rest): cache credentials ([#9620](https://github.com/googleapis/google-cloud-cpp/pull/9620))
```
Comment (dbolduc) [inline CHANGELOG.md:182]: If this is something `storage `used to do, I would delete the entry.

If not, should we move the entry under `storage`?

Comment (coryan) [reply CHANGELOG.md:176]: I will leave it here for now.  Yes, only storage uses these right now, but that will eventually change.

File: CHANGELOG.md (hunk 1 above)
Comment (coryan) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (coryan) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 4 above)
Comment (coryan) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 5 above)
Comment (coryan) [reply CHANGELOG.md]: Done

**Review by scotthart — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9763)* status: 0 of 1 files reviewed, 7 unresolved discussions (waiting on @coryan, @dbolduc, and @scotthart)

___
*[`CHANGELOG.md` line 119 at r2](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9763#-NAtoqnzE-gK67datUU2:-NAtoqnzE-gK67datUU3:blzci4l) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/dd220a07d835f749a549c33a4542f47f7c2035a9/CHANGELOG.md#L119)):*
> ```Markdown
> </details>
> 
> ## v2.2.0 - TBD
> ```

Update to 2022-09

**Review by devbww — COMMENTED** [review]
Update the `## v2.2.0 - TBD` header.

**Comment by devbww** [comment]
> Update the `## v2.2.0 - TBD` header.

Oops ... duplicate of Scott's comment.

**Review by dbolduc — APPROVED** [review]

**Review by coryan — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9763)* status: 0 of 1 files reviewed, 7 unresolved discussions (waiting on @dbolduc and @scotthart)

___
*[`CHANGELOG.md` line 119 at r2](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9763#-NAtoqnzE-gK67datUU2:-NAtrqlvCnoMrijpWZgK:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/dd220a07d835f749a549c33a4542f47f7c2035a9/CHANGELOG.md#L119)):*
<details><summary><i>Previously, scotthart (Scott Hart) wrote…</i></summary><blockquote>

Update to 2022-09
</blockquote></details>

Done.

File: CHANGELOG.md [hunk 7]
```diff
@@ -116,7 +116,7 @@ may elicit a deprecation warning. See [#9086] for more details.
 
 </details>
 
-## v2.2.0 - TBD
+## v2.2.0 - 2022-09
```
Comment (dbolduc) [inline CHANGELOG.md:119]: we should also add the `## v2.3.0 - TBD` right?

Comment (dbolduc) [reply CHANGELOG.md:119]: you could also do it in the version bump PR.

Comment (coryan) [reply CHANGELOG.md:119]: I will do that in the PR bumping new versions.

**Review by scotthart — APPROVED** [review]

**Pull request merged**